items/REAPER_MASK.json

```diff
@@ -4,7 +4,7 @@
   "displayname": "§6Reaper Mask",
   "clickcommand": "viewrecipe",
   "damage": 3,
-  "nbttag": "{HideFlags:254,SkullOwner:{Id:\"f4d7f301-cd28-353c-8027-8c617257b861\",Properties:{textures:[0:{Value:\"eyJ0aW1lc3RhbXAiOjE1NjgwMzY3MjYwNjYsInByb2ZpbGVJZCI6IjQxZDNhYmMyZDc0OTQwMGM5MDkwZDU0MzRkMDM4MzFiIiwicHJvZmlsZU5hbWUiOiJNZWdha2xvb24iLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzFmYzAxODQ0NzNmZTg4MmQyODk1Y2U3Y2JjODE5N2JkNDBmZjcwYmYxMGQzNzQ1ZGU5N2I2YzJhOWM1ZmM3OGYifX19\"}]}},display:{Lore:[0:\"§7Health: §a+150\",1:\"§7Defense: §a+100\",2:\"§7Intelligence: §a+100\",3:\" §8[§7❤§8] §8[§8✎§8]\",4:\"\",5:\"§6Ability: Disgusting Healing\",6:\"§7While wearing:\",7:\"§7- 2x healing from all sources\",8:\"§7- 2x Zombies from Reaper Scythe\",9:\"§7- §7Upgrades Zombie Armor\",10:\"§7effect to trigger on all\",11:\"§7hits\",12:\"\",13:\"§7§8This item can be reforged!\",14:\"§c☠ §5Requires Zombie LVL 7\",15:\"§6§lLEGENDARY HELMET\"],Name:\"§6Reaper Mask\"},ExtraAttributes:{modifier:\"unpleasant\",id:\"REAPER_MASK\"}}",
+  "nbttag": "{HideFlags:254,SkullOwner:{Id:\"f4d7f301-cd28-353c-8027-8c617257b861\",Properties:{textures:[0:{Value:\"eyJ0aW1lc3RhbXAiOjE1NjgwMzY3MjYwNjYsInByb2ZpbGVJZCI6IjQxZDNhYmMyZDc0OTQwMGM5MDkwZDU0MzRkMDM4MzFiIiwicHJvZmlsZU5hbWUiOiJNZWdha2xvb24iLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzFmYzAxODQ0NzNmZTg4MmQyODk1Y2U3Y2JjODE5N2JkNDBmZjcwYmYxMGQzNzQ1ZGU5N2I2YzJhOWM1ZmM3OGYifX19\"}]}},display:{Lore:[0:\"§7Health: §a+150\",1:\"§7Defense: §a+100\",2:\"§7Intelligence: §a+100\",3:\" §8[§7❤§8] §8[§8✎§8]\",4:\"\",5:\"§6Ability: Disgusting Healing\",6:\"§7While wearing:\",7:\"§7- 2x healing from all sources\",8:\"§7- 2x Zombies from Reaper Scythe\",9:\"§7- Allows you to store and\",10:\"§7summon an additional §e2 §7souls\",11:\"§7in their necromancer weapon.\",12:\"§7- §7Upgrades Zombie Armor\",13:\"§7effect to trigger on all\",14:\"§7hits\",15:\"\",16:\"§7§8This item can be reforged!\",17:\"§c☠ §5Requires Zombie LVL 7\",18:\"§6§lLEGENDARY HELMET\"],Name:\"§6Reaper Mask\"},ExtraAttributes:{modifier:\"unpleasant\",id:\"REAPER_MASK\"}}",
```

Comment:
There is a modified of unpleasant there just FYI

Reply:
uff how tf did it came there